Sources/XcbeautifyLib/String+Beautify.swift

```diff
@@ -196,6 +197,13 @@ extension String {
         return _colored ? "[\(target.f.Cyan)] \("Processing".s.Bold) \(filename)" : "[\(target)] Processing \(filename)"
     }
 
+    private func formatCompileCommand(pattern: Pattern) -> String? {
+        let groups = capturedGroups(with: pattern)
+        let leading = "Compile Command For"
+        let command = groups[0]
+        guard let filePath = groups.last else { return nil }
+        return "\(leading.s.Bold) \(filePath):\n\(command)"
```

Comment:
What would this print for your example? Have you taken a look at how xcpretty parse this?

Reply:
For the same file, sample.txt,

`cat sample.txt | xcpretty`
 will output:
Compiling Stdafx.cpp

`cat sample.txt | xcbeautify` 
will output:
[TargetName] Compiling Stdafx.cpp

`cat sample.txt | xcbeautify -V 3`
will output:
```
[TargetName] Compiling Stdafx.cpp
Compile Command For /Path/to/Stdafx.cpp:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c++ -target blablabla ....
```

Reply:
I really appreciate you took time to implement verbose feature, but I think we can just ignore the detail compile command output, the same way `xcpretty` does, since it's the purpose of using a beatifier tool to begin with. Users can always use the `tee` command to pipe the raw xcodebuild output somewhere and if they want more detailed output.

Reply:
I totally understand what you say and agree xcbeautify should keep simple :) By "ignoring the detail compile command", do you mean return nil from formatCompileCommand?

Reply:
That's right.

Reply:
I've updated code. Please review.